File: Test/test_cases/tests/Test3.java
import java.io.*;
class Test
{
  public int test() {
    Buyer x1=new Buyer(100);
    if(x1.getCash()==100)
    {
      x1.setCash(20);
      if(x1.getCash()==20)
      {
        return 1;
      }
      else
      {
        return 0;
      }
    }
    else
    {
      return 0;
    }
  }
}

File: execution_nodes/submissions/test_cases/cpp/tests/Test3.cpp
#include "Buyer.hpp"

using namespace std;

int test()
{
    Buyer buyer(100);
    if(buyer.getCash()==100)
    {
      buyer.setCash(20);
      if(buyer.getCash()==20)
      {
        return 1;
      }
      else
      {
        return 0;
      }
    }
    else
    {
      return 0;
    }
}
